app/javascript/components/Pages/ListEmotions.js

```diff
@@ -48,6 +68,11 @@ function ListEmotions(props) {
 
   const mix_up = index => ( index - 6 * (Math.ceil( index / 6 ) - 1 )) * 6 - (Math.ceil ( index / 6 ) - 1 ) - 1
 
+  console.log( 'response:', response)
+  console.log( 'emotions:', emotions)
+  console.log( 'timePeriod:', timePeriod)
+  console.log( 'curUserId:', curUserId)
```

Comment:
If these aren't needed for something specific it's better to remove console logs.

Reply:
Yes, I agree with you, next time I will pay attention to it.